### ISSUE TITLE
Specto Fork issues

### ISSUE THREAD

**Issue by tonytouch518** [mrknow/filmkodi#413]
Help please..I uninstalled Specto and tried to reinstall it because I wasn't able to choose from "in theaters"or look up certain movies.and now it says Add on is incompatible or has been marked broken in repository. Dependencies not met. I tried to reinstall and am unable to. Any help is much appreciated.

**Comment by xxcriticxx** [comment]
try to install URLResolver and specto Themespak
or 
install older version and it will update to the latest version

**Comment by pknoyes** [comment]
I can no longer use the download function in Specto.  Can you help?

**Comment by xxcriticxx** [comment]
that never worked correctly for me

**Comment by pknoyes** [comment]
It's been working fine, but since I did a Kodi reinstall, and reinstalled Specto, I can't get it to work.  That is on the TV with an Android box.  I loaded Kodi into my Windows laptop and everything works fine, including Specto Fork and the downloading.   So until I get over this hump, I take the laptop into the TV room and plug it into the TV with an HDMI cable and watch the shows I've downloaded in Specto on the laptop.
I think a big part of the problem is not being able to bring up the "C" prompt on the Android box, and select a target for the downloads.
